src/registry_schemas/version.py

```diff
@@ -23,4 +23,4 @@
 """
 
 
-__version__ = '2.18.28'  # pylint: disable=invalid-name
+__version__ = '2.18.29'  # pylint: disable=invalid-name
```

Comment:
Should update to `2.18.30`

Reply:
there is already a `2.18.29` released, and... there might be more coming out.
https://github.com/bcgov/business-schemas/releases